src/datadog.ts

```diff
@@ -25,6 +25,14 @@ export interface ServiceCheck {
   host_name: string
 }
 
+export interface Log {
+  ddsource: string
+  ddtags: string
```

Comment:
Any reason why for logs we should use a comma separated string instead of an array of strings?
I have no strong opinions but I'd like to keep the config consistent across the different configs

Reply:
@masci 
I followed the example of this API document. This says the type of field ddtags is string (comma separated value).
https://docs.datadoghq.com/api/latest/logs/#send-logs

But, of course, the consistency is important. Could you judge which is better?

Reply:
I see, thanks for the link.

In this case I think it'is better to stay consistent with the Datadog API, let's use your approach 👍